lib/app/views/languages/languages.erb

```diff
@@ -5,6 +5,12 @@
 	<% if active %>
 		<article class="article-block">
 			<div class="article-contents">
+				<% if docs[:docs][:about].empty? == false %>
+					<h2 class="header"><%= language %> Information</h2>
```

Comment:
Yes, did not like *Ruby Information* either. At the moment couldn't think of *About*. Thanks!